### ISSUE TITLE
Flickity v3 quasi-released

### ISSUE THREAD

**Issue by desandro** [metafizzy/flickity#1210]
Heads up: This weekend I released a new major version of Flickity, v3.0.0. The code is up on GitHub, but there is still plenty more work to be done: updating add-ons like `flickity-fullscreen` and then updating the [Flickity documentation site](https://flickity.metafizzy.co).

I recommend you continue to use Flickity v2 and its supported add-ons.

Best,
Dave

**Comment by S-n-d** [comment]
Just wanted to say **THANK YOU** for all the hard work. ❤️

**Comment by yaodingyd** [comment]
You are the best, Dave!

**Comment by derWebdesigner** [comment]
@desandro Is there any update on this? Can't wait to try the new version since v2 seems to have issues with Safari for a few months. Would happily pay for the new version again but some info on the timeline would be great because of the current issues I would have to switch to SwiperJS otherwise.

**Comment by RyanofWoods** [comment]
In my opinion, I think it would be beneficial if these quasi releases would instead use some beta release semantics by using `rc`, `alpha` and `beta` in the version tag, and by not setting the NPM package's default to these versions. This would allow users to test out these versions if they wish and that developers/dependabot don't bump to unstable quasi releases unknowingly. The fact that 3.0.0 is a quasi release is quite hidden, as it's in the issues section, and there's no changelog for the release yet.

Our team thought 3.0.0 was a normal release but was not documented yet, so we had to pull Q/A on it, where it failed 🙁 

What do you all think? 🙂

**Comment by badfeather** [comment]
@desandro Quick question - If I want to test flickity 3.0 and flickity-fade 2.0 out, what's the method to import them? Do you have any demos up?

**Comment by coreyworrell** [comment]
Yeah, very frustrating to not be able to update npm dependencies either as normal, have to remember each time to update everything except flickity. It's been in this state for almost a year now.

**Comment by razaanstha** [comment]
@desandro. any updates on this; the npm library is installing the quasi release of v3 instead of the stable release v2? Also the documentation is still not yet released for the v2?

**Comment by mylastore** [comment]
How to install version 3 latest build ?

**Comment by Webmaster-Siag** [comment]
> How to install version 3 latest build ?

Put this in the head:
```
<link rel="stylesheet" href="https://unpkg.com/flickity@3/dist/flickity.min.css">
  <link rel="stylesheet" href="https://unpkg.com/flickity-fullscreen@2/fullscreen.css">
```

And this in the body:
```
  <script src="https://unpkg.com/flickity@3/dist/flickity.pkgd.min.js"></script>
  <script src="https://unpkg.com/flickity-fullscreen@2/fullscreen.js"></script>
```
